{"uuid":"83e22571-2f3f-41cf-b46e-c97de63e1ff4","historyId":"e82c212770c88e0815b067ff880aed17","fullName":"DB.DBTest.SelectAll","labels":[{"name":"package","value":"DB.DBTest"},{"name":"testClass","value":"DB.DBTest"},{"name":"testMethod","value":"SelectAll"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Automatisation"},{"name":"subSuite","value":"DB.DBTest"},{"name":"host","value":"DESKTOP-M7P85M0"},{"name":"thread","value":"10928@DESKTOP-M7P85M0.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"SelectAll","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"org.postgresql.util.PSQLException: FATAL: password authentication failed for user \"habrpguser;\"","trace":"java.lang.RuntimeException: org.postgresql.util.PSQLException: FATAL: password authentication failed for user \"habrpguser;\"\r\n\tat DB.ConnectToDriver.connect(ConnectToDriver.java:30)\r\n\tat DB.DBSteps.connectDB(DBSteps.java:13)\r\n\tat DB.DBTest.SelectAll(DBTest.java:10)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:578)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\r\n\tat org.testng.TestRunner.run(TestRunner.java:505)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\r\n\tat org.testng.TestNG.run(TestNG.java:1017)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: org.postgresql.util.PSQLException: FATAL: password authentication failed for user \"habrpguser;\"\r\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.doAuthentication(ConnectionFactoryImpl.java:693)\r\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.tryConnect(ConnectionFactoryImpl.java:203)\r\n\tat org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:258)\r\n\tat org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:54)\r\n\tat org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:263)\r\n\tat org.postgresql.Driver.makeConnection(Driver.java:443)\r\n\tat org.postgresql.Driver.connect(Driver.java:297)\r\n\tat java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)\r\n\tat java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)\r\n\tat DB.ConnectToDriver.connect(ConnectToDriver.java:27)\r\n\t... 24 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1700253803544,"stop":1700253803840}